### DISCUSSION TITLE
Rewrite lcoale maping

### DISCUSSION THREAD

**Discussion by utsav0209** [i18next/next-i18next#1516] [Q&A] [answered]
Is it possible to map multiple languages to same translation file? 

e.g.
```js
{
   "fr-EN": "fr",
   "fr-US": "fr",
}
```

I am using next.js@10 and LTS of next-i18next.

**Comment by isaachinman** [comment, marked as answer]
I believe you're looking for locale fallbacks. Please check previous issues like #1465 for more information. In short, you should be able to achieve this purely through your i18next configuration.

**Reply by utsav0209** [reply]
@isaachinman the issue mentions at the end that `fallbackLang` is not working as expected, I tried it myself and I can confirm its not working with. Do you have any idea if there is any another way to do this?
